src/ifaddrs.rs

```diff
@@ -144,4 +176,21 @@ mod tests {
     fn test_getifaddrs() {
         let _ = getifaddrs();
     }
+
+    #[test]
+    fn test_getifaddrs_netmask_correct() {
+        let addrs = getifaddrs().unwrap();
+        for iface in addrs {
+            let sock = if let Some(sock) = iface.netmask {
+                sock
+            } else {
+                continue
+            };
+            if sock.family() == Some(crate::sys::socket::AddressFamily::Inet) {
+                let _ = sock.as_sockaddr_in().unwrap();
+                return;
+            }
+        }
+        panic!("No address?");
```

Comment:
This will panic on an IPv6-only system.  Don't do that.

Reply:
It shouldn't? There's localhost. But I can always do Inet || Inet6.